apps/web/src/actions/evaluations/connect.ts

```diff
@@ -15,33 +10,16 @@ export const connectEvaluationsAction = withProject
   .input(
     z.object({
       documentUuid: z.string(),
-      commitUuid: z.string(),
       templateIds: z.array(z.number()),
       evaluationUuids: z.array(z.string()),
     }),
   )
   .handler(async ({ ctx, input }) => {
-    const selectedTemplates = await filterEvaluationTemplatesById(
-      input.templateIds,
-    ).then((r) => r.unwrap())
-    const scope = new EvaluationsRepository(ctx.workspace.id)
-    const selectedEvaluations = await scope
-      .filterByUuids(input.evaluationUuids)
-      .then((r) => r.unwrap())
-
-    const documentsScope = new DocumentVersionsRepository(ctx.workspace.id)
-    const document = await documentsScope
-      .getDocumentAtCommit({
-        projectId: ctx.project.id,
-        commitUuid: input.commitUuid,
-        documentUuid: input.documentUuid,
-      })
-      .then((r) => r.unwrap())
-
     const connectedEvaluations = await connectEvaluations({
-      document,
-      templates: selectedTemplates,
-      evaluations: selectedEvaluations,
+      workspace: ctx.workspace,
+      documentUuid: input.documentUuid,
+      evaluationUuids: input.evaluationUuids,
+      templateIds: input.templateIds,
```

Comment:
Services should always receive model instances, not identifiers, and tenancy checks should live at the boundaries of the application (actions, in this case) 🙏🏼 This is mainly to ensure developers always implement tenancy when adding public endpoint.

 I know you are passing the workspace and probably implementing tenancy through that internally but it breaks the pattern and sets us up for the next dev to implement a service without a proper tenancy check.